src/GafferArnold/IECoreArnoldPreview/ShaderAlgo.cpp

```diff
@@ -123,7 +123,7 @@ std::vector<AtNode *> convert( const IECore::ObjectVector *shaderNetwork, const
 		{
 			if( boost::starts_with( shader->getType(), "osl:" ) )
 			{
-				nodeType = "osl_shader";
+				nodeType = "osl";
```

Comment:
I did note that - it seems to have been there from the start of the prototype osl_shader node as well.

It seems like given OSL's support in the backend for on the fly compiled code, it would be reasonable to expect the renderer backend to expose something like this.